Source/Renderer/ShaderSource.js

```diff
@@ -224,6 +242,12 @@ define([
             }
         }
 
+        // GLSLModernizer inserts its own layout qualifiers
+        // at this position in the source
+        if (context.webgl2) {
+            result += '#define OUTPUT_DECLARATION\n\n';
```

Comment:
Minor, but I would suggest to use a token that is a bit more unique, e.g., prefix with `CZM_` like we do elsewhere.